src/appMain/hmi_capabilities.json

```diff
@@ -462,8 +466,252 @@
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"FRONT_LEFT_TURN_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"FRONT_RIGHT_TURN_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"REAR_LEFT_FOG_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"REAR_RIGHT_FOG_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"REAR_LEFT_TAIL_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"REAR_RIGHT_TAIL_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"REAR_LEFT_BREAK_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"REAR_RIGHT_BREAK_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"REAR_LEFT_TURN_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"REAR_RIGHT_TURN_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
+                                 "name":"REAR_REGISTRATION_PLATE_LIGHT",
+                                 "sRGBColorSpaceAvailable":true
+                              },
+                              {
+                                 "densityAvailable":true,
```

Comment:
it is OK to have "densityAvailable":true and "sRGBColorSpaceAvailable":true for a group of lights. But suggest to set them to false.
Similarly for other light groups.